.github/workflows/actionlint.yml

```
@@ -0,0 +1,13 @@
+name: Actionlint
+
+on:
+  # Runs on pull requests
+  pull_request:
+
+jobs:
+  actionlint:
+    name: Run actionlint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: it-at-m/lhm_actions/action-templates/actions/action-actionlint@a4dd26733adc7d71b10fbd744ef355c01e4b7040 # v1.0.8
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)